static/app/views/issueList/customViewsHeader.spec.tsx

```diff
@@ -265,6 +265,57 @@ describe('CustomViewsHeader', () => {
         })
       );
     });
+
+    it('initially selects a temporary tab if the viewId is a default viewId and the query is different', () => {
```

Comment:
Is this the behavior that we want? For saved tabs it keeps you on the same tab and displays unsaved changes so I'd expect it to work the same for an unsaved tab, right?

Reply:
Hmm yeah you're definitely right about this - redirecting to a temporary tab does conflict with what we would expect from any other tab. Will make that change now.